### PR TITLE
Fix enum switcher

### LINKED ISSUE #99: Usage of enum_switcher widgets results in warning spam
The repro from #88 applies almost exactly, it just didn't work as a reproducer for this issue because I hadn't enabled log printing. I've now updated the same repo to show this issue: https://github.com/jplatte/druid-enum-switcher-bug

Warnings look like this:

```
2022-01-10T17:47:50.088952Z  WARN event: druid::core: WidgetId(3) received an event (MouseMove(MouseEvent { pos: (254.9296875, 126.9453125), window_pos: (254.9296875, 126.9453125), buttons: MouseButtons(00000), mods: Modifiers(NUM_LOCK), count: 0, focus: false, button: None, wheel_delta: Vec2 { x: 0.0, y: 0.0 } })) without having been laid out. This likely indicates a missed call to set_origin.
```

and occur for every mouse movement within the window / `Switcher` (haven't tried whether it happens outside of the switcher if it doesn't occupy the whole window).

### PR DESCRIPTION
**Pull request by xarvic**
This PR fixes #99 by adding the missing call to set_origin in `PrismWrap`.

### CODE REVIEW

File: src/prism.rs [hunk 1]
```diff
@@ -272,9 +252,7 @@ impl<T: Data> Prism<Option<T>, T> for OptionSome {
     }
 }
 
-prism!(OptionNone: Option<String> => None);
```
Comment (jplatte) [inline src/prism.rs:275]: Thanks for reverting this weird thing as well, it tripped me up before.

**Review by jplatte — APPROVED** [review]
Thanks for the quick fix, it works!

**Review by derekdreery — APPROVED** [review]
Thanks!

File: src/prism.rs (hunk 1 above)
Comment (xarvic) [reply src/prism.rs:275]: Yeah. It was work in progress, but I never hat the time to finish it.

**Comment by xarvic** [comment]
And Thanks for maintaining this Repo, I hope I can help a bit :)

**Pull request merged**